### ISSUE TITLE
Web pages shows purchase complete but no payment actually sent 

### ISSUE THREAD

**Issue by ghost** [getlantern/lantern#4876]
Getting more info from the user 
https://github.com/getlantern/forum/issues/263#issuecomment-230093701

**Comment by myleshorton** [comment]
Yeah this was the bug we were working on last week. How do you know payment
was not sent?

**Comment by p3rcya** [comment]
@myleshorton User check the alipay statement. I'll get more info. It might be the user doesn't complete the process correctly

**Comment by myleshorton** [comment]
I bet this on our end - thanks and fyi @uaalto

**Comment by fffw** [comment]
Update: User purchases Pro on a Vivo x6d Android 5.1 phone. The WebView turns to complete empty after the green checkmark disappeared. I doubt if the `/purchase` call is made in this case. If WebView is closed, the UI won't be fully empty. @atavism

**Comment by p3rcya** [comment]
The user also said when Lantern is off, he is able to complete the purchase

**Comment by uaalto** [comment]
Is this what he gets?:

* Page shows "payment complete"
* Seems like Pro was purchased
* It actually isn't, and as soon as the user refreshes the screen it will display the free interface again.
